docs/data-tunnel/05-getting-started.md

```diff
@@ -0,0 +1,61 @@
+# Getting Started with Data Tunnel
+
+This guide provides a quick reference on how to use Data Tunnel to send price data to a destination.
+
+## Step 1: Prepare the Consumer Contract
+
+To send packet data from BandChain to the destination chain, the client must implement a contract based on the destination chain and the selected route type. The implementation details vary for each integration method. For example, refer to [IBC Hook Integration - Implementing a WASM Contract](./route-types/03-ibc-hook.md#implementing-a-wasm-contract).
+
+## Step 2: Create a Tunnel
+
+Each tunnel route has specific requirements for tunnel creation. For example, creating an IBC Hook tunnel requires specifying the `[channel-id]` and `[destination-contract-address]`.
```

Comment:
I think this page we explain common parameters and adds links to each route type (on create section) so they can follow to create. think it will be easier because at this point they still don't understand each route.

Reply:
instead of give ibc-hook as example.